### DISCUSSION TITLE
Incorrect Customer-Mail-Adress

### DISCUSSION THREAD

**Discussion by Getting-Good-de** [woocommerce/woocommerce#52683] [Checkout Flow]
Falsche Kunden-Mail-Adresse.
Das Feld sollte leer sein, damit die Kunden dort ihre eigene Mail Adresse eintragen können.
Stattdessen steht da vorausgefüllt MEINE  Mail-Adresse! Ich bin doch aber der Verkäufer, daher macht das keinen Sinn. Die Mail-Adresse läßt sich leider aber auch nicht bearbeiten oder löschen. So was kann ich machen, damit meine Mail Adresse aus diesem Feld verschwindet?
LG Silvia

Incorrect customer email address.
The field should be empty so that customers can enter their own email address there.
Instead it says MY email address pre-filled! But I'm the seller, so that doesn't make any sense. Unfortunately, the email address cannot be edited or deleted. What can I do to make my email address disappear from this field?
regards Silvia

**Comment by mikejolley** [comment]
Hi,

Have you tried in a different browser while logged out? In the editor you'll see your email address because the field defaults to the email address of the current logged in user. A new customer should not see this. 

Thanks
